app/reducers/optional.js

```diff
@@ -1,7 +0,0 @@
-//this is a simple pass thru that offers an extension point
```

Comment:
yeah there's more going on in the service than i'd imagined ... guess I should have looked rather than assume. I still need time to _digest_ functional programming so give me a sec to fully grok your setup.